packages/moonstone/styles/colors-light.less

```diff
@@ -78,7 +78,8 @@
 // ContextualPopup
 // ---------------------------------------
 @moon-contextual-popup-bg-color: @moon-bg-color;
-@moon-contextual-popup-border-color: fade(@moon-black, 50%);
+@moon-contextual-popup-border-color: #bbb;
```

Comment:
Any insight on why we need a new color here (`#bbbbbb`) vs using `@moon-gray` (`#a6a6a6`)?

Reply:
#bbbbb is a new color for border colors. #a6a6a6 is close, but we decided to keep it separate.